src/server/iframe.html.js

```diff
@@ -3,6 +3,11 @@ export default function (headHtml) {
     <!DOCTYPE html>
     <html>
       <head>
+        <script>
+          if (window.parent !== window) {
```

Comment:
Actually, this could be completely omitted and instead added to the `head.html` by Storybook users as needed.